src/conf_mode/nat_cgnat.py

```diff
@@ -224,15 +221,31 @@ def get_config(config=None):
         with_recursive_defaults=True,
     )
 
-    if conf.exists(base) and is_node_changed(conf, base + ['pool']):
-        config.update({'delete_conntrack_entries': {}})
+    effective_config = conf.get_config_dict(
+        base,
+        get_first_key=True,
+        key_mangling=('-', '_'),
+        no_tag_node_value_mangle=True,
+        effective=True,
+    )
+
+    # Check if the pool configuration has changed
+    if not conf.exists(base) or is_node_changed(conf, base + ['pool']):
+        config['delete_conntrack_entries'] = {}
+
+    # add running config
+    if effective_config:
+        config['effective'] = effective_config
+
+    if not conf.exists(base):
+        config['remove'] = {}
```

Comment:
It's called `deleted` in most of the other places, please adjust.

```python
def verify(ethernet):
    if 'deleted' in ethernet:
        return None
```

Reply:
Done!